images/build/php/prereqs/installPrereqs.sh

```diff
@@ -39,6 +39,10 @@ sed -e 's/stretch/buster/g' /etc/apt/sources.list > /etc/apt/sources.list.d/bust
 	echo 'Pin: release n=buster';
 	echo 'Pin-Priority: 990';
 } > /etc/apt/preferences.d/argon2-buster;
-apt-get update;
-apt-get install -y --no-install-recommends libsodium-dev;
+apt-get update
+apt-get install -y --no-install-recommends \
+    libsodium-dev
+rm -rf /var/lib/apt/lists/*
```

Comment:
Fixes https://github.com/microsoft/Oryx/issues/703

Reply:
libsodium-dev is something very important for php ... can you please run nightly builds as well to make sure this is not adding other issues in the php side

Reply:
Sure, sounds good.